tests/Unit/Evolution/Systems/ScalarWave/Test_EnergyDensity.cpp

```diff
@@ -20,6 +20,11 @@
 #include "Utilities/TMPL.hpp"
 
 namespace {
+template <size_t SpatialDim, typename DataType>
+void test_compute_item_in_databox() noexcept {
```

Comment:
you're adding a test for the _simple_ tag, which should go in `Test_Tags.cpp`

Reply:
you still need to remove the `test_compute_item_in_databox` from the commit "Add EnergyDensity simple tag and test"